lib/cisco_node_utils/bgp_af.rb

```diff
@@ -101,17 +101,22 @@ def set_args_keys(hash={}) # rubocop:disable Style/AccessorMethodName
     # Client to client (Getter/Setter/Default)
     #
     def client_to_client
+      return nil if platform == :ios_xr && @vrf != 'default'
       state = config_get('bgp_af', 'client_to_client', @get_args)
       state ? true : false
     end
 
     def client_to_client=(state)
+      fail Cisco::UnsupportedError.new('bgp_af', 'client-to-client', 'set',
```

Comment:
Not sure the '/' concatenated code is worth it. Maybe just:
if platform == :ios_xr && @vrf != 'default'
  fail code
end

?